common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -33,6 +33,57 @@
 from .helpers import BaseDiscussionMixin
 
 
+mathjax_body = """I wrote up my solution to Problem 7.57 in the text in the hope that it is helpful. If you would like
```

Comment:
How about `THREAD_CONTENT_WITH_LATEX`?

For reference: https://www.python.org/dev/peps/pep-0008/#naming-conventions